Documentation/building/unix/dependencies.md

```diff
@@ -155,3 +172,79 @@ Suggested `brew install` commands:
 If any program is still not found, try to ensure it's linked via:
 
     brew link <package name>
+
+
+Note: Apple provided `git` seems to be old and user might experience problems during tool detection reporting
+that `git` is missing:
+
+```
+Checking autoconf                                    [FOUND 2.69]
+Checking automake                                    [FOUND 1.15]
+Checking cmake                                       [FOUND 3.10.2]
+Checking git                                         [MISSING]
+Checking make                                        [FOUND 3.81]
+Checking mingw-w64                                   [MISSING]
+Checking ninja                                       [FOUND 1.10.0]
+Checking p7zip                                       [FOUND 16.02]
+```
+
+and that could lead to further errors like:
```

Comment:
The messages below aren't really errors, they're git progress messages output to `stderr` which `xaparepare` will mark in red. The messages are harmless, if annoying.

Reply:
Would it be possible to add a sink/adapter which doesn't mark this particular "stderr output" in red?  So that *real* errors are more obvious?

Reply:
It would. I'll work on it.